datajunction-clients/python/datajunction/client.py

```diff
@@ -17,7 +16,9 @@ class DJClient(_internal.DJClient):
     """
 
     #
-    # List
+    # List objects:
+    # - basic: namespaces, dimensions, metrics, cubes
+    # - TODO (maybe): sources, transforms, nodes...
     #
     def list_namespaces(self, prefix: Optional[str] = None) -> List[str]:
```

Comment:
fwiw, I think `list_sources` would be useful. I've found myself wanting that while I use the client

Reply:
Sounds good. Will do.

Reply:
@shangyian let me close this PR and add this in a follow up. I'd like to refresh the Readme and Notebook first.

Reply:
Sounds great! Thanks @agorajek